events.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_sqs_queue" "this" {
+  count = var.enable_sqs_monitoring ? 1 : 0
+
+  name_prefix = "prefect-worker-"
+}
+
+resource "aws_sqs_queue_policy" "this" {
```

Comment:
I haven't seen the pattern before of naming the resource `this`, is that common?